Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX_test.py

```diff
@@ -0,0 +1,423 @@
+from CrowdStrikeFalconX import Client, test_module,\
+    upload_file_command, send_uploaded_file_to_sandbox_analysis_command, send_url_to_sandbox_analysis_command,\
+    get_full_report_command, get_report_summary_command, get_analysis_status_command, download_ioc_command, \
+    check_quota_status_command, find_sandbox_reports_command, find_submission_id_command
+import pytest
+
+UPLOAD_FILE_ARGS = {
```

Comment:
should not we split the responses/expected_results to a separate file? it will make the test file much more readable.